src/modules/dashboard/Dashboard.tsx

```diff
@@ -35,6 +42,44 @@ const Dashboard: FC<DashboardProps> = () => {
         flexDirection="column"
         gap="spacing-md"
       >
+        {successMessage && (
+          <Box margin="spacing-sm spacing-none spacing-none spacing-none">
+            <Alert
+              variant="success"
+              heading={successMessage}
+            />
+          </Box>
+        )}
+
+        {errorMessage && (
+          <Box margin="spacing-sm spacing-none spacing-none spacing-none">
+            <Alert
+              variant="error"
+              heading={errorMessage}
+            />
+          </Box>
+        )}
+
+        {isWalletConnected && (
+          <SocialHandles
+            errorMessage={errorMessage}
+            setErrorMessage={setErrorMessage}
+            successMessage={successMessage}
+            setSuccessMessage={setSuccessMessage}
+            padding={{ ml: 'spacing-md spacing-sm', initial: 'spacing-md' }}
+            claimButton={
+              <Link to={'/points/activity'}>
+                <Button
+                  aria-label="Claim"
+                  size="small"
+                  variant="tertiary"
+                >
+                  Claim
+                </Button>
+              </Link>
```

Comment:
the claim button should be changed to claimed and disabled after the activities are claimed.